packages/experiments/src/components/Persona/Vertical/VerticalPersona.styles.ts

```diff
@@ -1,9 +1,10 @@
-import { IVerticalPersonaComponent, IVerticalPersonaStylesReturnType } from './VerticalPersona.types';
+import { IVerticalPersonaComponent, IVerticalPersonaStylesReturnType, IVerticalPersonaTokenReturnType } from './VerticalPersona.types';
 
-export const VerticalPersonaTokens: IVerticalPersonaComponent['tokens'] = (props, theme) => {
+export const VerticalPersonaTokens: IVerticalPersonaComponent['tokens'] = (props, theme): IVerticalPersonaTokenReturnType => {
   return {
     verticalPersonaWidth: 122,
-    font: theme.fonts.large,
+    // TODO: typing exposed this error, ask Mark what it should be?
+    // font: theme.fonts.large,
```

Comment:
@Markionium FYI these typing helpers exposed this error. I'm not sure what should be here (if anything)

Reply:
Thanks @JasonGore i'll fix it.